docs/de/README.md

```diff
@@ -78,9 +78,9 @@ curl https://get.wasmer.io -sSfL | sh
   
   * <a href="https://crates.io/crates/wasmer-cli/">Cargo</a>
 
-    _Note: All the available
-    features are described in the [`wasmer-cli`
-    crate docs](https://github.com/wasmerio/wasmer/tree/main/lib/cli/README.md)_
+    _Notiz: Alle verfügbaren Merkmale sind in der [`wasmer-cli`
```

Comment:
Translated "features" to "Merkmale" here, since I think "feature" is less of a technical term.